src/blueprints/tv2/value-objects/tv2-action.ts

```diff
@@ -60,6 +61,13 @@ export interface Tv2RemoteAction extends Tv2PartAction {
   }
 }
 
+export interface Tv2RecallLastRemoteAsNextAction extends Tv2RemoteAction {
+  type: PartActionType.INSERT_PART_AS_NEXT,
+  metadata: Tv2RemoteAction['metadata'] & {
+    actionSubtype: Tv2ActionSubtype.RECALL_LAST_REMOTE,
+  }
```

Comment:
Couldn't this just be:
```
metadata: {
  actionSubtype: Tv2ActionSubtype:RECALL_LAST_REMOTE
}
```

You set the value of `remoteNumber` to `-1` with the comment `// Is not used for this action`. 
If the attributes of `Tv2RemoteAction['metada']` is not used for this Action, then why do the type gymnastics to include them?

Reply:
`contentType` is in that case missing. But you are right. I have made it extend from Tv2Part instead.